autoload/ale/definition.vim

```diff
@@ -17,6 +20,18 @@ function! ale#definition#ClearLSPData() abort
     let s:go_to_definition_map = {}
 endfunction
 
+function! ale#definition#UpdateTagStack() abort
+    let l:should_update_tagstack = exists('*gettagstack') && exists('*settagstack') && g:ale_update_tagstack
+
+    if l:should_update_tagstack
+        let l:from = [bufnr('%'), line('.'), col('.'), 0]
+        let l:tagname = expand('<cword>')
```

Comment:
You might have to wait until the buffer is opened first, and grab the word second.

Reply:
Same as above - the values I'm pushing on to the stack are based on the old location.  So I don't think I need to wait for the buffer to open; assuming I've understood this comment correctly?